src/main/webapp/index.jsp

```diff
@@ -42,11 +43,19 @@
         Through our site, you will be able to connect with the world through online chat!
       </p>
       <ul>
-        <li><a href="/login">Login</a> to get started.</li>
-        <li>Go to the <a href="/conversations">conversations</a> page to
+        <% if(request.getSession().getAttribute("user") != null){ %>
+          <li>Go to the <a href="/conversations">conversations</a> page to
             create or join a conversation.</li>
-        <li>View the <a href="/about.jsp">about</a> page to learn more about the
+          <li>View the <a href="/about.jsp">about</a> page to learn more about the
             project.</li>
+          <li>View the <a href="/profile.jsp">My profile</a> page to view your profile.</li>
+        <% } else{ %>
+          <li><a href="/login">Login</a> to get started.</li>
+          <li>Go to the <a href="/conversations">conversations</a> page to
```

Comment:
It looks like this list item and the "about" item below it are exactly the same as in the block above. Could you pull these out of the if-statement so that they don't have to be repeated?

Reply:
Would this be like creating a navbar.jsp file and setting the format up there, and including the navbar in each file?

Reply:
I was actually just thinking you could do something like:

if user != null {
  View the My profile page.
} else {
  Log in to get started.
}
Go to the conversations page.
View the about page.

But I agree that pulling the whole navbar situation into a separate jsp that can be included is a great way to avoid having to update every single page every time you change something!  Doesn't necessarily have to be part of this PR, but it would be a good thing to do at some point.